### ISSUE TITLE
REAPR crashes after running a while

### ISSUE THREAD

**Issue by rkdavies** [skinwalker-ranch-insiders/reapr#1]
```
 File "/usr/local/lib/python3.10/site-packages/h2/connection.py", line 224, in process_input
    func, target_state = self._transitions[(self.state, input_)]
KeyError: (<ConnectionState.CLOSED: 3>, <ConnectionInputs.RECV_PING: 14>)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/h2/connection.py", line 228, in process_input
    raise ProtocolError(
h2.exceptions.ProtocolError: Invalid input ConnectionInputs.RECV_PING in state ConnectionState.CLOSED

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/httpcore/_sync/http2.py", line 144, in handle_request
    raise LocalProtocolError(exc)  # pragma: nocover
httpcore.LocalProtocolError: Invalid input ConnectionInputs.RECV_PING in state ConnectionState.CLOSED

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/httpx/_transports/default.py", line 77, in map_httpcore_exceptions
    raise mapped_exc(message) from exc
httpx.LocalProtocolError: Invalid input ConnectionInputs.RECV_PING in state ConnectionState.CLOSED
```

**Comment by rkdavies** [comment]
Seems connection drops briefly to YT and Connection Closes.

Need to add a check for this, wait a minute or so and then retry before dying.

**Comment by rkdavies** [comment]
Fixed in [bfbd7dd659](https://github.com/skinwalker-ranch-insiders/reapr/tree/bfbd7dd6590d785336de0778667c20162e05d95f) #bfbd7dd659
